ads/ads.py

```diff
@@ -741,7 +852,9 @@ def up(args):
     _add_verbose_arg(parser)
     _add_services_arg(parser)
     parsed_args = parser.parse_args(args)
-    ads = _load_or_die()
+    ads = _load_or_die(use_cache=ALWAYS
```

Comment:
It seems gnarly that this code knows about the special case of _resolve_selectors where there's no selector. It's also somewhat wrong - if the default service is specified, then the cache could be used.

Is there anyway to refactor this so the code that calls load doesn't have to know the details of how resolve_selectors works?